### DISCUSSION TITLE
Re-interview node command through mqtt ?

### DISCUSSION THREAD

**Discussion by davidwaze** [zwave-js/zwave-js-ui#2757] [Q&A] [answered]
Hello,

I'm having an issue with Qubino ZMNHCD Flush Shutter Z-Wave module when i'm having a power outage that my roller shutter does not indicate level position and power consumption anymore, of course i can control it and it works, it's only reporting part which is not OK.
The only solution i found is to reinterview device and works again. (but by doing it manually by zwavejs-ui for my 10 roer shutter is not the best solution).
I'm suspecting a bug from the firmware of the device (1.1) that i can't update (didn't find any firmware for this, if you have why not).

Except if you find another solution, my idea was to reinterview device automatically by a mqtt command through my home automation software. 

Is it possible to realize this ?

Thanks.

**Comment by robertsLando** [comment, marked as answer]
@davidwaze I don't think it's the best way to fix this

Anyway it si possible, check [zwave-apis](https://zwave-js.github.io/zwave-js-ui/#/guide/mqtt?id=z-wave-apis)

In your case the api is `refreshInfo` and the only arg needed is the nodeID so the payload will look like:

```
{
 "args": [ 12 ]
}
```

Replace `12` with your nodeId

Also `refreshCCValues` could work.

To fix the root cause of your problem I need to ask @AlCalzone what he suggest

**Comment by davidwaze** [comment]
Hello

I'm aware it's really not the best solution...

I forgot to mention that of course informations are refreshed correctly if I request a refresh manually.

The power consumption is changing all the time while opening or closing roller shutter I can't imagine to refresh it all the time from my home automation. To open/close I would also imagine to have an automatic refresh after 30 sec for example. But in any case I think that requesting something from my home automation box is the not the good idea, only having incoming informations from zwave-js.

The best thing would be to solve the root problem but really don't know how ... and happen only after a power outage when device is restarting and don't push those information's anymore. 

Thanks for your feedback.

**Comment by AlCalzone** [comment]
If the device reports power but not position, this is very likely an issue with the device. I'm not sure what the interview does that fixes it - cannot be the associations since the power reports are still coming in.

You should probably reach out to Qubino to try and get this fixed. The only problem: to my knowledge they require you to mail in devices for a firmware update.

**Comment by davidwaze** [comment]
It doesn't report position AND power after power outage.

I get them only by requesting them manually or need to reinterview to have pushed information again to have a standard behavior.

**Comment by AlCalzone** [comment]
Oh then I misunderstood. So either it is forgetting the route to the controller or the lifeline association.
You could try if healing the node (giving it a new route) fixes the issue.

My point about contacting the manufacturer support still stands though.

**Comment by davidwaze** [comment]
Healing does not work already tried 😔

I'm not sure to have any support from them for device which are 6-7 years old... The firmware upgrade is the solution I think.

No workaround possible by launching interview from mqtt by I don't know how ...

**Comment by AlCalzone** [comment]
@robertsLando Posted the workaround above

**Comment by davidwaze** [comment]
It works like a charm, thanks again for your (very very very) quick support :-)

I had some difficulties to make it work as i was using api refreshinfo with i instead of I :-)

**Comment by robertsLando** [comment]
Glad that fixed the issue :)
